### ISSUE TITLE
Invisible Skeletons?

### ISSUE THREAD

**Issue by Justacode** [MushirMickeyJoe/Relatively-Improved-Default#62]
This is more for the Fresh Animations x Barely Default, since every time I add it with the 1.20.1 fresh animations, it causes all skeletons' skins invisible and just leaves their bow floating in the air. I'm playing on Forge 47.1.3 v1.20.1 with the Entity Model Features and Entity Texture Features mods instead of Optifine. I tried switching the packs every way possible after the right way didn't work but nothing fixes it. Barely Default V9.25
Barely Default x Fresh Animations (2 latest versions) 
Forge 47.1.3 v1.20.1

**Comment by MushirMickeyJoe** [comment]
Hey man thanks for your comment, have you tried the pack without Forge, just the default installation with only OptiFine?
That version worked for me, and is what I uploaded as V9.25 of Barely Default.
I suspect the OptiFine replacement mods aren't up to the task. Could you message them and send them my pack? I will do the same

**Comment by MushirMickeyJoe** [comment]
I wanted to add that in update V9.24 and V9.25 I revamped the my skeleton system to use layers, so I can randomize (biome dependant, shown [here](https://www.youtube.com/watch?v=9USB5JIYG-8&list=PLO6GPyTLxi6tpgNiGLoKuTFc-dggL3w-O&index=1) ) the base layer with the armour layers.

I do this by assigning a texture to a limb, instead of the entire skeleton using one texture.

This OptiFine CEM feature isn't well known - it's not even implemented in BlockBench yet - so it's possible these replacement mods are missing that specific feature.

**Comment by Justacode** [comment]
Hi, I've tried the pack without forge and it does work, the only thing is Optifine for that version of forge is being difficult and I wanted to play it with mods so I was looking for a way around it. Would I just be able to use the V9.23 then since it was the updates before the skeleton change? Also, huge fan of the pack btw, it's awesome. Oh and I'll be sure to inform the optifine replacements too.

**Comment by MushirMickeyJoe** [comment]
Yeah man play with V9.23 I guess till the mod creator updates. The zombies also use that system so I hope nothing else breaks for you...

You could also go into the assets/Minecraft/optifine/cem folder and remove the models for skeletons

Thanks for the love 💙

**Comment by Justacode** [comment]
Oh I found a work around the skeletons but now the redstone golem is screwin up. I'll just have fun with it. Thanks for the advice!
